M2/Macaulay2/m2/set.m2

```diff
@@ -183,9 +183,6 @@ fixedOperators = unique toList fixedOperators
 allOperators = unique toList allOperators
 
 protect Flexible
-protect Binary
-protect Prefix
-protect Postfix
```

Comment:
How come Flexible is kept here?

Reply:
Tangentially, I think a syntax like:
```m2
otimes = ⊗ = new BinaryOperator from symbol ⊗
otimes.precedence = precedence symbol *
```
would be nice, though perhaps for flexible operators it's a bit more tricky.

Reply:
dunno I didn't touch this

Reply:
the problem with using the syntax `new ... from symbol ...` is that there is no room for options, and I'm not sure how to implement this `.precedence` thing.

Reply:
> How come Flexible is kept here?
dunno I didn't touch this

Reply:
should I have moved `Flexible` too? I'm not even clear what it means.

Reply:
> dunno I didn't touch this

You removed `protect Binary` and `Prefix` and `Postfix` but kept `Flexible` here. Why?

Reply:
well, besides what I wrote above, the practical reason was that I didn't need `Flexible`.